locale/en_US/submission.xml

```diff
@@ -44,6 +44,7 @@
 	<message key="submission.confirmSubmit">Are you sure you wish to submit this manuscript to the press?</message>
 	<message key="submission.metadata">Metadata</message>
 	<message key="submission.supportingAgencies">Supporting Agencies</message>
+	<message key="submission.grid.titleColumn">Author; Book Title</message.
```

Comment:
at the end of the lib/pkp/locale/en_US/grid.xml file, you can create another section like  <!-- Suibmission list grids -->
and add that locale key, without the specific object name, of course.